CGPs/0024.md

```diff
@@ -38,13 +37,19 @@ Fill out the following template for each transaction in the proposal
 
 ## Verification
 
-An explanation of how voters can verify that this CGP does what it intends to do. Can be left as “TODO” until the proposal is made. Include things like CLI commands to run and pointers to code.
+With the following CLI command
+
+```bash
+celocli governance:view --proposalID XX
+```
+TODO: Replace XX by the actual proposal ID once submitted.
```

Comment:
Otherwise, if you already know what the proposal ID is going to before submitting it, I'm fine with just modifying `XX` to reflect the ID Number right before submission. 

What I want to avoid is having any outstanding `TODOs` lines prior to submission on-chain as it might indicate the draft isn't finalized.